application/repositories/interfaces.py

```diff
@@ -1,6 +1,10 @@
-from abc import ABC
+from abc import ABC, abstractmethod
+from typing import List
 
+from application.entities.room_entity import RoomEntity
 
-class BaseRepository(ABC):
-    def __init__(self, db_repo) -> None:
-        self.db_repo = db_repo
+
+class GetAllRooms(ABC):
```

Comment:
The interface is not a class! At this point I added an interface that represents the `GetAllRooms()` abstraction, and that's what you need to implement.